swabbie-redis/src/main/kotlin/com/netflix/spinnaker/swabbie/redis/RedisWorkQueue.kt

```diff
@@ -40,6 +43,7 @@ class RedisWorkQueue(
 
   init {
     log.info("Using ${javaClass.simpleName}")
+    this.track()
```

Comment:
What about this , shouldn't we get rid of this now ?
https://github.com/spinnaker/swabbie/blob/master/swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/work/WorkQueueManager.kt#L118